src/Common/src/Interop/Unix/libcurl/Interop.libcurl_slist.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+using size_t = System.UInt64;
```

Comment:
Right now we're only targeting 64-bit.  By the time we support 32-bit as well (assuming we do), all of this should be smoothed over by our native shims.